modules/local/nf-core-modified/sourmash/gather/main.nf

```diff
@@ -16,7 +16,8 @@ process SOURMASH_GATHER {
     val save_prefetch_csv
 
     output:
-    tuple val(meta), path('*.csv.gz')             , optional:true, emit: result
+    // the "result" output was modified from the nf-core module to .csv from .csv.gz
```

Comment:
Does nf-core usually require all output files to be `.gz`? I have wondered that about some modules...

Reply:
yes.